components/postByAuthorMapping.tsx

```diff
@@ -1,43 +1,72 @@
 import React from "react";
-import AuthorDescription from "./author-description"
+import AuthorDescription from "./author-description";
 import Image from "next/image";
+import { animated, useInView, easings } from "@react-spring/web";
 
-const PostByAuthorMapping = ({ filteredPosts ,Content}) => {
+function Node({ node }) {
+  const [ref, springStyles] = useInView(
+    () => ({
+      from: {
+        opacity: 0,
+      },
+      to: {
+        opacity: 100,
+      },
+      config: {
+        duration: 500,
+        delay: 100,
+        easing: easings.easeInCubic,
+      },
+    }),
+    {
+      rootMargin: "-200px 0px",
+    }
+  );
+  return (
+    <animated.li className="mb-8" ref={ref} style={springStyles}>
+      <a href={`/blog/${node.categories.edges[0].node.name}/${node.slug}`}>
```

Comment:
since we are using next you can use <Link> instead of <a>

Reply:
I'm sorry, I could not understand, can you please explain this change?

Reply:
you can use Link tag instead of anchor tag